apps/web/components/ListViewCard/ListViewCard.tsx

```diff
@@ -125,21 +125,27 @@ export const ListViewCard = ({
           </Box>
         )}
         <Box paddingTop={3} width="full" height="full">
-          <Box display={'flex'} alignItems={'flexEnd'} height="full">
+          <Box
+            display={'flex'}
```

Comment:
display="flex" is 2 fewer characters (nitpick, doesn't really matter, do what you want with this information)